### PR TITLE
Add staking program and mesh program to xc-admin UI

### PR DESCRIPTION
**Pull request by guibescos**
This will be useful to review some proposals to transfer authorities to governance and also for adding other Anchor programs in the future.

### CODE REVIEW

File: governance/xc_admin/packages/xc_admin_common/src/multisig_transaction/MessageBufferMultisigInstruction.ts [hunk 1]
```diff
@@ -2,50 +2,76 @@ import {
...
-export class MessageBufferMultisigInstruction implements MultisigInstruction {
-  readonly program = MultisigInstructionProgram.MessageBuffer;
+export const MESH_PROGRAM_ID = new PublicKey(
+  "SMPLVC8MxZ5Bf5EfF7PaMiTCxoBAcmkbM2vkrvMK8ho"
+);
+export const STAKING_PROGRAM_ID = new PublicKey(
+  "pytS9TjG1qyAZypk7n8rw8gfW9sUaqqYyMhJQ4E7JCQ"
+);
+
+export class AnchorMultisigInstruction implements MultisigInstruction {
```
Comment (guibescos) [inline governance/xc_admin/packages/xc_admin_common/src/multisig_transaction/MessageBufferMultisigInstruction.ts:22]: Small refactor here to make it easier to support multiple Anchor programs

File: governance/xc_admin/packages/xc_admin_common/src/multisig_transaction/MessageBufferMultisigInstruction.ts [hunk 2]
```diff
@@ -2,50 +2,76 @@ import {
   MultisigInstruction,
   MultisigInstructionProgram,
   UNRECOGNIZED_INSTRUCTION,
+  UnrecognizedProgram,
 } from ".";
 import { AnchorAccounts, resolveAccountNames } from "./anchor";
 import messageBuffer from "message_buffer/idl/message_buffer.json";
```
Comment (m30m) [inline governance/xc_admin/packages/xc_admin_common/src/multisig_transaction/MessageBufferMultisigInstruction.ts:8]: maybe rename this `messageBufferIdl` to be consistent

File: governance/xc_admin/packages/xc_admin_common/src/multisig_transaction/MessageBufferMultisigInstruction.ts [hunk 3]
```diff
@@ -2,50 +2,76 @@ import {
...
+        program = MultisigInstructionProgram.Mesh;
+        break;
+      case STAKING_PROGRAM_ID.toBase58():
+        idl = stakingIdl as Idl;
+        program = MultisigInstructionProgram.Staking;
+        break;
+      default:
+        return UnrecognizedProgram.fromTransactionInstruction(instruction);
+    }
+    const instructionCoder = new BorshCoder(idl as Idl).instruction;
```
Comment (m30m) [inline governance/xc_admin/packages/xc_admin_common/src/multisig_transaction/MessageBufferMultisigInstruction.ts:61]: do we need another `as Idl` here too?

File: governance/xc_admin/packages/xc_admin_common/src/multisig_transaction/MessageBufferMultisigInstruction.ts [hunk 4]
```diff
@@ -2,50 +2,76 @@ import {
...
+      return new AnchorMultisigInstruction(
+        program,
         deserializedData.name,
         deserializedData.data,
-        resolveAccountNames(
-          messageBuffer as Idl,
-          deserializedData.name,
-          instruction
-        )
+        resolveAccountNames(idl as Idl, deserializedData.name, instruction)
```
Comment (m30m) [inline governance/xc_admin/packages/xc_admin_common/src/multisig_transaction/MessageBufferMultisigInstruction.ts:70]: do we need another `as Idl` here too?

**Pull request merged**